cves/CVE-2011-1088.yml

```diff
@@ -208,4 +228,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This was mostly a coding bug, as the annotations were simply forgetten about
+    for version 7.0.0 of Tomcat. Security permissions were enforced if they were
+    defined in the XML, however they were ignored as annotations. This shows that
```

Comment:
I am unsure what you mean by "ignored as annotations." Is there more specific information that could be said about how this bug occurred?

Reply:
"Security permissions were enforced if they were defined in the XML, however they were ignored as annotations."
Are you saying that all XML defined security permissions were ignored as annotations?

Reply:
Reworded parts of this to make it easier to understand